Misc/NEWS.d/next/Windows/2019-10-05-05-50-58.bpo-38344.scr2LO.rst

```diff
@@ -0,0 +1 @@
+Fix usage of _OLD_VIRTUAL_PATH in activate.bat.  Do not use parenthesis in the if statement.  Instead of if/else, use if defined/if not defined on separate lines.
```

Comment:
The NEWS item doesn't need to be this detailed - it'll link back to the bug, which links to the PR, which links to the changes for anyone interested. Simply saying "Fix error message in activate.bat" would be fine.